script/Arcueid/Day 2/02_00_ARC02_1D.txt

```diff
@@ -61,7 +63,8 @@ Thanks to how sophisticated the internet has become, the knowledge gap between g
 [sha:86febd0eba3130afe40aa8fa9d4f4ba0d0beaeec]{
 -- Page 117, Offset 2907.
 -- 確固として積み上げた物量の差はあるものの、生活していく上ではイーブンになったように見えてしまう。
-Although there is a difference in the number of things we have accumulated, we seem to have evened out in terms of lifestyles.
+Even though some people have accumulated more material goods than others,
+ we seem to have evened out in terms of lifestyles.
 }
 [sha:5bb0e0fcbd88c331effae5fa14f52a1c92aa39fd]{
 -- Page 118, Offset 2908.
```

Comment:
2917
>But hey, for how much I've been going on about this pretentious analysis, it's ultimately just my own thoughts.

Reply:
Bunch of stuff in this fill needs to be consistent, and correct.
2909
>The world is small these days. Even elementary school students know how closed in our society has become.

2916
>I think that's what the depressing sense of being trapped in classrooms, schools, and cities is all about.

2918
>I don't feel like I'm trapped.

2919
>行き詰まりも倦怠感も、社会への不満もまるでない。
>Nor do I feel like I'm closed in, fatigued, or dissatisfied with society. Not at all.